Source/Atomic/Graphics/AnimatedModel.h

```diff
@@ -99,6 +99,8 @@ class ATOMIC_API AnimatedModel : public StaticModel
 
     /// Return skeleton.
     Skeleton& GetSkeleton() { return skeleton_; }
+    /// Return the node of a skeleton bone (for script access)
+    Node* GetSkeletonBoneNode(const String& boneName);

```

Comment:
When modifying core Urho sources, it is good to enclose new methods in a // ATOMIC BEGIN and // ATOMIC END block : https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/Atomic/Graphics/Animation.h#L181

I also have been doing this in the implementation side: https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/Atomic/Graphics/Animation.cpp#L398

... and trying to be consistent when modifying code, also trying to block/comment.  

This greatly helps when pulling in updates, I can get this in post